cassandra-diagnostics-connector21/pom.xml

```diff
@@ -0,0 +1,106 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.smartcat</groupId>
+    <artifactId>cassandra-diagnostics</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>cassandra-diagnostics-connector21</artifactId>
+  <description>Cassandra Diagnostics Connector for Cassandra 2.1.x.</description>
+  
+    <properties>
+        <version.cassandra.diagnostics>1.0.0</version.cassandra.diagnostics>
+        <version.junit>4.12</version.junit>
+        <version.bytebuddy>1.3.12</version.bytebuddy>
+        <version.aopalliance>1.0</version.aopalliance>
+        <version.cassandra>2.1.13</version.cassandra>
+        <version.plugin.compiler>3.5.1</version.plugin.compiler>
+        <version.plugin.shade>2.4.3</version.plugin.shade>
+        <version.powermock>1.6.5</version.powermock>
+        <version.spring_instrument>4.2.6.RELEASE</version.spring_instrument>
+        <version.plugin.surfire>2.19.1</version.plugin.surfire>
+        <version.hector.core>2.0-0</version.hector.core>
+        <version.cassandra.driver>3.0.0</version.cassandra.driver>
```

Comment:
The driver is used only for the integration test in order to generate queries from Java.